### PR TITLE
Add documentation for known GKE issue detailed in issue 515

### PR DESCRIPTION
**Pull request by thefirstofthe300**
Addresses documentation requested in https://github.com/kedacore/keda/issues/515#issuecomment-678974693

### CODE REVIEW

File: content/troubleshooting/gke-external-metrics.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  - "external.metrics.k8s.io"
+  resources:
+  - "*"
+  verbs:
+  - list
+  - get
+  - watch
+```
+
+The GKE team is currently working on a fix that they expect to have out in version <= 1.16.13.
```
Comment (zroubalik) [inline content/troubleshooting/gke-external-metrics.md:29]: Shouldn't be here >= 1.16.13? Or am I misunderstanding this?

Comment (thefirstofthe300) [reply content/troubleshooting/gke-external-metrics.md:29]: Nope. Brain fart on my part.

File: content/troubleshooting/gke-external-metrics.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
++++
+title = "Why does GKE 1.16 fail to fetch external metrics?"
++++
+
+If you are receiving the following error:
```
Comment (tomkerkhove) [inline content/troubleshooting/gke-external-metrics.md:5]:
```suggestion
If you are running Google Kubernetes Engine (GKE) version 1.16, and are receiving the following error:
```

File: content/troubleshooting/gke-external-metrics.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
++++
+title = "Why does GKE 1.16 fail to fetch external metrics?"
```
Comment (tomkerkhove) [inline content/troubleshooting/gke-external-metrics.md:2]: Let's use the full name for those who don't use GCP, maybe?
```suggestion
title = "Why does Google Kubernetes Engine (GKE) 1.16 fail to fetch external metrics?"
```

File: content/troubleshooting/gke-external-metrics.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
++++
+
+If you are receiving the following error:
+
+```
+unable to fetch metrics from external metrics API: <METRIC>.external.metrics.k8s.io is forbidden: User "system:vpa-recommender" cannot list resource "<METRIC>" in API group "external.metrics.k8s.io" in the namespace "<NAMESPACE>": RBAC: clusterrole.rbac.authorization.k8s.io "external-metrics-reader" not found
+```
+
+and are running GKE version 1.16, you are almost certainly running into a [known issue](https://issuetracker.google.com/issues/160597676). The workaround is to recreate the `external-metrics-reader` role using the following YAML:
```
Comment (tomkerkhove) [inline content/troubleshooting/gke-external-metrics.md:11]: Adding new line to optimize rendered docs:
```suggestion
You are almost certainly running into a [known issue](https://issuetracker.google.com/issues/160597676). 

The workaround is to recreate the `external-metrics-reader` role using the following YAML:
```

File: content/troubleshooting/gke-external-metrics.md (hunk 1 above)
Comment (tomkerkhove) [reply content/troubleshooting/gke-external-metrics.md:29]: This makes more sense indeed, just adding so we make sure it's changed.
```suggestion
The GKE team is currently working on a fix that they expect to have out in version >= 1.16.13.
```

**Review by tomkerkhove — APPROVED** [review]
LGTM, just added a few nits to improve the rendered docs.

**Comment by tomkerkhove** [comment]
Thanks @thefirstofthe300 !

**Pull request merged**